README.md

```diff
@@ -1932,7 +1996,7 @@ EXAMPLES
   $ mw autocomplete --refresh-cache
 ```
 
-_See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomplete/blob/v3.0.16/src/commands/autocomplete/index.ts)_
+_See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomplete/blob/v3.0.13/src/commands/autocomplete/index.ts)_
```

Comment:
Looks like you somehow DOWNgraded some of the dependencies. Since the `yarn.lock` is unmodified → did you perhaps forget to install the latest dependencies before generating the README?

Reply:
I was not aware of this, fixed with 5b1b3a4230e7dd732015d94c52e86267b50156bb